workflow/apm/services.mdx

```diff
@@ -0,0 +1,46 @@
+---
+title: Services
+sidebarTitle: Services
+---
+
```

Comment:
Recommend changing "this landing page" to "Drilling into any service from this list will display common metrics..."